src/parser/xasl_generation.c

```diff
@@ -718,7 +718,7 @@ pt_make_connect_by_proc (PARSER_CONTEXT * parser, PT_NODE * select_node, XASL_NO
   select_xasl->instnum_pred = pt_to_pred_expr_with_arg (parser, instnum_part, &flag);
   if (flag & PT_PRED_ARG_INSTNUM_CONTINUE)
     {
-      select_xasl->instnum_flag = XASL_INSTNUM_FLAG_SCAN_CONTINUE;
+      select_xasl->instnum_flag |= XASL_INSTNUM_FLAG_SCAN_CONTINUE;
     }
```

Comment:
this line looks same logic with add_instnum_predicate() in plan_generation.c. we may re-use it.
xasl_generation.c:718
xasl_generation.c:13998
xasl_generation.c:14140

because add_instnum_predicate() is a static function, it needs to be moved to xasl_generation.h and modified as general.
```
/* plan_generation.c (as-is) */
static XASL_NODE *
add_instnum_predicate (QO_ENV * env, XASL_NODE * xasl, PT_NODE * pred)

/* move to xasl_generation.h */
static XASL_NODE *
pt_to_instnum_pred(PARSER_CONTEXT * parser, XASL_NODE * xasl, PT_NODE * pred);
```

modify function signature
```
please refer plan_generation.c:683, 758, 931;
as-is:
xasl = add_instnum_predicate (env, xasl, instnum_pred);

to-be:
xasl = pt_to_instnum_pred (QO_ENV_PARSER (env), xasl, instnum_pred);
```

the following code can be refactored by calling the function.
```
xasl_generation.c:718
xasl_generation.c:13998
xasl_generation.c:14140
```

Reply:
thanks. I fixed it